src/components/Nav/TeamMenu.vue

```diff
@@ -230,6 +202,32 @@ export default {
             </v-list-item-subtitle>
           </v-list-item-content>
         </v-list-item>
+        <v-list-item :to="'/team/cloud-hooks'">
+          <v-list-item-avatar tile>
+            <i class="o-100 fad fa-clouds fa-2x" />
+          </v-list-item-avatar>
+          <v-list-item-content>
+            <v-list-item-title class="text-subtitle-1">
+              Cloud Hooks
+            </v-list-item-title>
+            <v-list-item-subtitle>
+              Create and modify team-wide Cloud Hooks
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <v-list-item :disabled="!isCloud" :to="'/team/tokens'">
+          <v-list-item-avatar tile>
+            <i class="o-100 fad fa-exchange-alt fa-2x" />
+          </v-list-item-avatar>
+          <v-list-item-content>
+            <v-list-item-title class="text-subtitle-1">
+              API Tokens
+            </v-list-item-title>
+            <v-list-item-subtitle>
+              Manage your team's API tokens
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
```

Comment:
Can we move these back up the list and alphabetize the rest of the list? These were meant to be ordered but that got lost at some point

Reply:
I moved these as I dislike having the deprecated API Tokens at the top of the list.  However we discussed and agreed that for consistency we should keep the list alphabetized- I'll update.